index.html

```diff
@@ -0,0 +1,126 @@
+          <img src="./image/apple_logo.png" alt="" class="nav-img"></button></div>
+      <div class="nav-spc">
+        <p></p>
+      </div>
+      <div class="nav-btn"><button type="button" name="button"><img src="./image/store.png" alt="" class="nav-img"></button></div>
+      <div class="nav-spc">
+        <p></p>
+      </div>
+      <div class="nav-btn"><button type="button" name="button"><img src="./image/Mac.png" alt="" class="nav-img"></button></div>
+      <div class="nav-spc">
+        <p></p>
+      </div>
+      <div class="nav-btn"><button type="button" name="button"><img src="./image/iPod.png" alt="" class="nav-img"></button></div>
+      <div class="nav-spc">
+        <p></p>
+      </div>
+      <div class="nav-btn"><button type="button" name="button"><img src="./image/iPhone.png" alt="" class="nav-img"></button></div>
+      <div class="nav-spc">
+        <p></p>
+      </div>
+      <div class="nav-btn"><button type="button" name="button"><img src="./image/iPad.png" alt="" class="nav-img"></button></div>
+      <div class="nav-spc">
+        <p></p>
+      </div>
+      <div class="nav-btn"><button type="button" name="button"><img src="./image/iTunes.png" alt="" class="nav-img"></button></div>
+      <div class="nav-spc">
+        <p></p>
+      </div>
+      <div class="nav-btn"><button type="button" name="button"><img src="./image/Support.png" alt="" class="nav-img"></button></div>
+      <div class="nav-spc">
+        <p></p>
+      </div>
+      <div class="nav-btn" id="search-box"><button style="height: 35px;width: 110px;" id="btn-r">
+          <input type="text" name="Search"></button></div>
+
+
+
+
+    </div>
+    <!-- navbar ends_ -->
+    <div class="main">
+      <!-- main page content starts-->
+      <div class="paragraph">
+        <a href="#"><img src="./image/your_verse_title_small.png" alt=""></a>
+        <a href="#"><img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_subhead_small.png"
+            width="980" height="82" alt=""></a><br>
+        <!-- <div id="under_l"></div> -->
+        <a href="#">
+          <div id="under_l"></div><a>
+
+
+      </div>
+      <!-- main page content ends_ -->
+    </div>
+  </div>
+
+
+
+  <div class="bottom">
+    <!-- bottom page content starts -->
+    <a href="#">
+      <div id="item1" class="item"><img src="./image/promo_iphone5s.jpg" alt=""> </div>
+    </a>
+    <a href="#">
+      <div id="item2" class="item"><img src="./image/iphone5c.jpg" alt=""></div>
+    </a>
+    <a href="#">
+      <div id="item3" class="item"><img src="./image/promo_30_years.jpg" alt=""></div>
+    </a>
+    <a href="#">
+      <div id="item4" class="item"><img src="./image/promo_supplier_responsibility.jpg" alt=""></div>
+    </a>
+    <!-- bottom page content ends_ -->
+  </div>
+
+  <footer>
+    <!-- footer section starts -->
+    <div class="left">
```

Comment:
You are using the `margin-left` property with percentage as units. The problem with that is that it's hard to know what to expect when the page is rendered in so many different devices. I'm attaching a screenshot of how the content it's displayed in a wide monitor with Full HD resolution (1920 x 1080)
![screen shot 2019-01-28 at 06 44 05](https://user-images.githubusercontent.com/14355495/51837187-3d5fc180-22c8-11e9-9c9f-d77d07fd3b5e.png)

See, the alignment there was broken